server/src/models/Task.ts

```diff
@@ -0,0 +1,13 @@
+class Task {
```

Comment:
Skoro juz mamy klase dla Task to moze dla text z tasku dodac settera i w nim jakas walidacje, ze np task nie moze byc pustym stringiem